components/nexus-repository/src/test/java/org/sonatype/nexus/repository/negativecache/NegativeCacheFacetImplTest.groovy

```diff
@@ -0,0 +1,278 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-2015 Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.repository.negativecache
+
+import com.google.common.collect.Lists
+import com.google.common.collect.Maps
+import net.sf.ehcache.Cache
+import net.sf.ehcache.CacheManager
+import net.sf.ehcache.Ehcache
+import net.sf.ehcache.Element
+import org.junit.Before
+import org.junit.Test
+import org.mockito.ArgumentCaptor
+import org.sonatype.nexus.repository.Repository
+import org.sonatype.nexus.repository.config.Configuration
+import org.sonatype.nexus.repository.http.HttpStatus
+import org.sonatype.nexus.repository.util.NestedAttributesMap
+import org.sonatype.nexus.repository.view.Status
+import org.sonatype.sisu.goodies.common.Time
+import org.sonatype.sisu.goodies.eventbus.EventBus
+import org.sonatype.sisu.litmus.testsupport.TestSupport
+
+import static net.sf.ehcache.Status.STATUS_ALIVE
+import static net.sf.ehcache.Status.STATUS_SHUTDOWN
+import static org.mockito.Matchers.any
+import static org.mockito.Mockito.mock
+import static org.mockito.Mockito.never
+import static org.mockito.Mockito.verify
+import static org.mockito.Mockito.when
+
+/**
+ * Tests for {@link NegativeCacheHandler}.
+ */
+class NegativeCacheFacetImplTest
+extends TestSupport
+{
+  private NegativeCacheFacetImpl underTest
+  private NegativeCacheKey key
+  private Status status
+  private CacheManager cacheManager
+  private Ehcache cache
+  private Repository repository
+  private Map attributes
+
+  @Before
+  void setUp() {
+    cacheManager = mock(CacheManager)
```

Comment:
@adreghiciu Is it possible to replace the mock() calls with @Mock annotations? Cleans up the boilerplate quite a bit if so, and TestSupport already calls initMocks so it should "just work"